pages/careers/_open-positions.html

```diff
@@ -193,25 +198,60 @@ <h4>Your Ideal Background</h4>
 
     <hr />
 
-    <h3>Senior Software Engineer<br />Senior Site Reliability Engineer</h3>
-    <p>
-      Just like our <a href="#software-engineer">Software Engineer</a> and
-      <a href="#site-reliability-engineer-sre">Site Reliability Engineer</a>
-      positions, but with the added expectation that you take more ownership,
-      get (complex) things done, and communicate even more effectively.
-    </p>
+    <a target="senior-product-designer"></a>
+    <h3>Senior Product Designer</h3>
 
-    <hr />
+    <p>Help us make Gruntwork delightful!</p>
+
+    <h4>What's Gruntwork?</h4>
+    <p>We give software teams a complete, world-class cloud infrastructure that lets them go to prod in days. Typically,
+       the process of launching prod infrastructure in the cloud is awful. It takes months of tedious work, hundreds of
+       thousands of dollars, and often results in poorly maintained code and anti-patterns in the end. We call our company
+       "Gruntwork" because we take away this Gruntwork for our clients so that they can focus on the parts of their software
+       that are unique to them.</p>
+    <p>We are 100% remote, ~15 people, profitable, growing (even during COVID!) and fully bootstrapped with no debt and
+       no investors. Our small company works with both startups and name companies like Toyota, Adobe, and the US House
+      of Representatives. We also <a href="#gruntwork-is-a-human-friendly-company">care a lot about humans.</a></p>
+
+    <h4>What's the opportunity?</h4>
+    <ul>
+      <li>
+        <strong>Product Design.</strong> We're building a SaaS platform that will become the primary customer experience of our product. Today, customers access our product as private GitHub repos plus a patchwork of links to training and documentation. Working with you, we envision a portal where customers login to discover and use all the elements of our existing product, brand it to themselves, and ultimately offer a UI-based way to learn, deploy and manage infrastructure in the cloud, all backed by infrastructure as code under the hood.
+      </li>
+      <li>
+        <strong>Branding and Marketing Website.</strong> We have an improved understanding of our business that we'd like to reflect in a new public marketing website. We'll look to you lead the design effort here.
```

Comment:
This is too narrow of a take. Focus on something more like, "You'd be the owner of Gruntwork's public face to the world. How do we tell the world what we do? How do we explain and showcase our highly technical products in a way that visitors can understand? How do we convert visitors to leads?"

Reply:
Good call. Fixed in 629e840.